packages/react-notion-x/src/components/eoi.tsx

```diff
@@ -1,8 +1,10 @@
 import * as React from 'react'
 import { Block } from 'notion-types'
+import { formatDistanceToNowStrict } from 'date-fns'
```

Comment:
I'm trying to avoid importing from `date-fns` in the main bundle because it adds quite a bit to the bundle size.